http/swagger.yml

```diff
@@ -12036,12 +12036,10 @@ components:
           - retention_policy
     DBRPs:
       properties:
-        notificationEndpoints:
+        content:
           type: array
           items:
             $ref: "#/components/schemas/DBRP"
-        links:
-          $ref: "#/components/schemas/Links"
```

Comment:
This seems standard in many other objects returned by the API, I think we should instead try to fix the implementation to fill it in.